lib/Consumer.d.ts

```diff
@@ -27,6 +27,11 @@ export declare type ConsumerOptions = {
      * by itself.
      */
     paused?: boolean;
+    /**
+     * The preferred MID for the Consumer. If not specified, a sequentially growing
+     * number will be assigned.
+     */
+    preferredMid?: string;
```

Comment:
Why preferredMid instead of mid? I mean, if given it's gonna always be used, right?

Reply:
yes, I assumed that it should always be used when specified. I could change the name to just `mid`. I've seen there are other optional parameters prefixed with preferred, so I thought it was some naming convention of optional parameters to override some default configuration.